README.md

```diff
@@ -1,10 +1,10 @@
-![](https://github.com/karam084/My_Portfolio_WebSite-)
+![]online version (https://karam084.github.io/portfolio_desktop_version/)
 
-# Portfolio Mobile version
+# Portfolio Desktop version
 
 > This is the personal Portfolio of Karam Fouad - Web Developer
 
-![screenshot](.\Microverse Projects\portfolio mobile version\screenshot\screenshot.docx)
+![screenshot](.\Microverse Projects\portfolio deskto version\screenshot\screenshot.docx)
```

Comment:
- [ ] There is an issue with the screenshot image in your README because it's not showing the image. I think you could use this [Markdown Cheat Sheet](https://www.markdownguide.org/cheat-sheet/) so you can fix this syntax issue in order to make your README file more professional :wink: